### PR TITLE
Update ownership of dotnet watch

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@mkArtakMSFT Who should be listed as an owner of BrowserRefresh?

FYI @ManishJayaswal  @marcpopMSFT

**Comment by marcpopMSFT** [comment]
@tmat what else is needed for merging this?

File: CODEOWNERS [hunk 1]
```diff
@@ -62,10 +62,11 @@
...
 # Area-Watch
-/src/Assets/TestProjects/Watch*/     @captainsafia @mkArtakMSFT
-/src/Tests/dotnet-watch.Tests/       @captainsafia @mkArtakMSFT
-/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @captainsafia @mkArtakMSFT
-/src/BuiltInTools/   @captainsafia @mkArtakMSFT
+/src/Assets/TestProjects/Watch*/     @tmat @mkArtakMSFT
+/src/Tests/dotnet-watch.Tests/       @tmat @mkArtakMSFT
+/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @mkArtakMSFT
+/src/BuiltInTools/*   @tmat @mkArtakMSFT
+/src/BuiltInTools/BrowserRefresh   @mkArtakMSFT
```
Comment (mkArtakMSFT) [inline CODEOWNERS:69]:
```suggestion
/src/BuiltInTools/BrowserRefresh   @MackinnonBuck @mkArtakMSFT
```

File: CODEOWNERS [hunk 2]
```diff
@@ -62,10 +62,11 @@
 /src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ClickOnce.targets @sujitnayak
 
 # Area-Watch
-/src/Assets/TestProjects/Watch*/     @captainsafia @mkArtakMSFT
-/src/Tests/dotnet-watch.Tests/       @captainsafia @mkArtakMSFT
-/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @captainsafia @mkArtakMSFT
-/src/BuiltInTools/   @captainsafia @mkArtakMSFT
+/src/Assets/TestProjects/Watch*/     @tmat @mkArtakMSFT
+/src/Tests/dotnet-watch.Tests/       @tmat @mkArtakMSFT
+/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @mkArtakMSFT
```
Comment (mkArtakMSFT) [inline CODEOWNERS:67]:
```suggestion
/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @MackinnonBuck @mkArtakMSFT
```

**Comment by tmat** [comment]
@arkalyanms FYI

**Review by mkArtakMSFT — APPROVED** [review]
Thanks @tmat !

**Review by arunchndr — APPROVED** [review]

**Pull request merged**